### DISCUSSION TITLE
Asterix dependency in dependency causes two yarn lock file entries after upgrading a dependency

### DISCUSSION THREAD

**Discussion by n1ru4l** [renovatebot/renovate#7958] [Request Help] [answered]
Reproduction: https://github.com/n1ru4l/graphql-live-query/pull/349

**Comment by rarkins** [comment, marked as answer]
I suggest you add `yarnDedupeFewer` or `yarnDedupeHighest` to [postUpdateOptions](https://docs.renovatebot.com/configuration-options/#postupdateoptions) in your config. It's not enabled by default because it could result in changes to lock files which are unrelated to the dependency being upgraded.
